firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -575,11 +575,12 @@ public static FetchResponse forBackendUpdatesFetched(
           lastFetchETag);
     }
 
-    public static FetchResponse forBackendHasNoUpdates(Date fetchTime) {
+    public static FetchResponse forBackendHasNoUpdates(
+        Date fetchTime, ConfigContainer fetchedConfigs) {
       return new FetchResponse(
           fetchTime,
           Status.BACKEND_HAS_NO_UPDATES,
-          /*fetchedConfigs=*/ null,
+          /*fetchedConfigs=*/ fetchedConfigs,
```

Comment:
I might add a comment that we're including the `fetchConfigs` because they include the template version (even though they don't include the config entries or metadata)